conf/supervisord-dev.conf

```diff
@@ -9,8 +9,8 @@ stderr_events_enabled=true
 stopsignal = KILL
 stopasgroup = true
 
-[program:refresh_blocklist]
-command=sh -c "while true; do bin/fetch-blocklist; sleep 60; done"
+[program:worker]
+command=newrelic-admin run-program celery -A checkmate.async.celery:app worker --loglevel=INFO
```

Comment:
This isn't exiting cleanly anymore when you <kbd><kbd>Ctrl</kbd> + <kbd>c</kbd></kbd> the `make dev` command in dev. It needs:

```
stopsignal = KILL
stopasgroup = true
```

at the bottom of this block. Like h has: https://github.com/hypothesis/h/blob/451f290edda91d12d2ed0df80fbc2a7794db3b73/conf/supervisord-dev.conf#L23-L24

Generally I think we want all processes (not just Celery workers) to have `stopsignal = KILL` and `stopasgroup = true`. Experimentation has showed that this means that the processes exit quickly and cleanly when <kbd><kbd>Ctrl</kbd> + <kbd>c</kbd></kbd>'s (without printing more stuff to your terminal after your prompt has been drawn) and processes aren't left running after.

Testing it now I can reliably stop `make dev` cleanly with `stopsignal = KILL` and `stopasgroup = true` and without them it reliably fails to stop cleanly, which is the same as I found in the past with the other apps.